fastly/client.go

```diff
@@ -333,7 +359,11 @@ func (c *Client) RequestForm(verb, p string, i interface{}, ro *RequestOptions)
 	if err != nil {
 		return nil, err
 	}
+
 	body := v.Encode()
+	if ro.HealthCheckHeaders {
```

Comment:
ℹ️ To reduce the blast radius of this type of manual serialisation I've locked it down to just the CREATE and UPDATE API endpoints for health checks.